### DISCUSSION TITLE
New Command to add

### DISCUSSION THREAD

**Discussion by Respawn00** [zfbx/zdiscord#56] [Ideas]
Add a command called /note [id] [Message] -- Allows admins to add notes to players accounts -- Admin  only command

/notes [id] - Show the admin the players notes  -- Admin  only command

/hmh (how many hours) [id] -- Allows admins to check players hours that have had on the game -- Admin only command

/givecar [id] [model] -- For QBCore gives player the car

And if there is anyway when a new player joins the server they get a code and need to verify using discord and if you can make that happen make a new command  - Example bellow -- Only gives them a code when they join the server for the first time

Example Code: 1041 then they need to go into the discord verify channel and do /verify 1041 and it links their ingame account with their discord so admins can do /p2d (player ID to Discord) [id] then it sends a message saying there discord account

**Comment by zfbx** [comment]
Quite the list of commands..

/notes would require a db table which I don't really want to try and explain adding for people (unless there already is one for qb?)

/hmh would also require a table and a persistent loop updating a players table +1 every 60 seconds 

/givecar is already in as /vehicle give

as for the last 2 I think that's in txadmin? but the original purpose of zdiscord was to be a whitelist where you'd vet them in your discord server beforehand and give them a whitelisted role

**Reply by Respawn00** [reply]
Is there a command to unban someone?

**Reply by Respawn00** [reply]
And for the notes command the notes could just stay on the discord account and not the persons in the game account so its like dynos notes system

**Reply by zfbx** [reply]
Sadly, nope, as there isn't a hook to unban in qb either, you have to delete the ban from the database directly.

**Reply by zfbx** [reply]
> And for the notes command the notes could just stay on the discord account and not the persons in the game account so its like dynos notes system

That also requires a database. why not just use dyno for the notes since it seems to have the feature you want already?

**Reply by Respawn00** [reply]
it would have been easy just to use 1 discord bot but its fine 

Your discord bot is great

**Reply by zfbx** [reply]
> 

Well yes and no. zdiscord is dependent on fivem being up so it's not great as an overall bot for things outside of fivem. it's better to have a dedicated discord bot that has moderation tools and such which would be too much for fivem to handle

**Reply by Respawn00** [reply]
can i change the / to !

**Reply by zfbx** [reply]
nope. slash commands are built into discord. to use chat commands you'd need to rewrite a lot of stuff and use the messageCreate event instead of interactionCreate. the old eris branch does that but it's very out of date and fairly poorly written
